torch/_dynamo/skipfiles.py

```diff
@@ -124,6 +125,78 @@ def _module_dir(m: types.ModuleType):
 FILENAME_ALLOWLIST |= {torch.optim._functional.__file__}
 FILENAME_ALLOWLIST |= {torch.utils._foreach_utils.__file__}
 
+if torch.distributed.is_available():
```

Comment:
a thought for maybe later.. at what point do we go from allowlisting a bunch of stuff in pt-d to inverting and only having a disallow list?

Reply:
No idea, probably never? The allow-list is always going to be controlled, whereas disallow list puts an onus on creation that I don't think we can ever impose.